etc/kayobe/environments/ci-multinode/inventory/group_vars/compute/lvm.yml

```diff
@@ -7,25 +7,25 @@ stackhpc_lvm_group_rootvg_lvs_extra:
 # StackHPC LVM Logical Volume (LV) configuration.
 
 # StackHPC LVM lv_swap LV size.
-stackhpc_lvm_lv_swap_size: 1g
+stackhpc_lvm_lv_swap_size: 500m
 
 # StackHPC LVM lv_root LV size.
-stackhpc_lvm_lv_root_size: 10g
+stackhpc_lvm_lv_root_size: 5g
 
 # StackHPC LVM lv_tmp LV size.
-stackhpc_lvm_lv_tmp_size: 10g
+stackhpc_lvm_lv_tmp_size: 5g
 
 # StackHPC LVM lv_var LV size.
-stackhpc_lvm_lv_var_size: 20g
+stackhpc_lvm_lv_var_size: 2g
```

Comment:
That's quite a drop!

Reply:
I'm not exactly sure why it was so large in the first place to be honest. Not a lot goes on in there from what I can see